api/src/controllers/resumable-uploads.ts

```diff
@@ -0,0 +1,93 @@
+import { Router, type Application } from 'express';
+import { ItemsService } from '../services/index.js';
+import { getSchema } from '../utils/get-schema.js';
+import { scheduleSynchronizedJob, validateCron } from '../utils/schedule.js';
+import { getTusServer, getTusStore } from '../services/tus/index.js';
+import { AuthorizationService } from '../services/authorization.js';
+import asyncHandler from '../utils/async-handler.js';
+import type { PermissionsAction } from '@directus/types';
+import { ForbiddenError } from '@directus/errors';
+import { RESUMABLE_UPLOADS } from '../constants.js';
+
+const mapAction = (method: string): PermissionsAction => {
+	switch (method) {
+		case 'POST':
+			return 'create';
+		case 'PATCH':
+			return 'update';
+		case 'DELETE':
+			return 'delete';
+		default:
+			return 'read';
+	}
+};
+
+const checkFileAccess = () =>
+	asyncHandler(async (req, _res, next) => {
+		const auth = new AuthorizationService({
+			accountability: req.accountability,
+			schema: req.schema,
+		});
+
+		if (!req.accountability?.admin) {
```

Comment:
Since we're checking the negative here without an else explicitly checking `null`/`undefined` on accountability and `false` on admin feels overly verbose to me 😬